common/lib/common-utils/src/bufferNode.ts

```diff
@@ -31,11 +31,7 @@ export type IsoBuffer = Buffer;
  */
 export function Uint8ArrayToString(arr: Uint8Array, encoding?: string): string {
     // Make this check because Buffer.from(arr) will always do a buffer copy
-    if (Buffer.isBuffer(arr)) {
-        return arr.toString(encoding);
-    } else {
-        return Buffer.from(arr).toString(encoding);
-    }
+    return Buffer.isBuffer(arr) ? arr.toString(encoding) : Buffer.from(arr).toString(encoding);
```

Comment:
Maybe in this particular case, but in general I think this produces more readable (and simpler) code. In cases where it really doesn't it can be ignored.